Makefile

```diff
@@ -38,9 +38,9 @@ uitest: ## 🧪 Run the ui tests in headless mode
 	@echo -e "\e[34m$@\e[0m" || true
 	@cd tests/integration/ui && npm install && npx cypress run --env ADMIN_WEBSITE_NAME=$(ADMIN_WEBSITE_NAME),FRONTEND_WEBSITE_NAME=$(FRONTEND_WEBSITE_NAME)
 
-build-frontend: ## 🏗️ Build the Frontend webapp
+unittest-frontend: ## 🏗️ Unit test the Frontend webapp
```

Comment:
Did you mean to replace `build-frontend` with this? It might be worth having both `build` and `unittest` tasks for the front end.

Reply:
I replaced it because it followed the same pattern as we have for the python code - we don't have an explicit "build" command, we only have a unit-test. 

I thought the only reason we had the build-frontend was because there weren't any tests to run. Do you see any benefit in keeping the build-frontend?

Reply:
lol, my brain is super-wired to java. Ok, I don't feel very strongly on the subject so I don't mind reinstating the build-frontend command, but I will leave the ci command to call the unittest-frontend command directly.